DQMOffline/LumiZCounting/BuildFile.xml

```diff
@@ -0,0 +1,10 @@
+<use name="FWCore/Framework"/>
+<use name="DataFormats/MuonReco"/>
+<use name="DataFormats/HLTReco"/>
+<use name="TrackingTools/IPTools"/>
+<use name="boost"/>
+<use name="root"/>
+<flags CXXFLAGS="-g -Wall"/>
```

Comment:
Please remove this. If you want to use "-g" or other flags locally, please use `export USER_CXXFLAGS="-ggdb"`.
But don't put them into module scope!